packages/core/src/Banner.js

```diff
@@ -2,48 +2,57 @@ import React from 'react'
 import Box from './Box'
 import Flex from './Flex'
 import Text from './Text'
-import Icon from './Icon'
 import CloseButton from './CloseButton'
-import Heading from './Heading'
 import PropTypes from 'prop-types'
 import styled, { withTheme } from 'styled-components'
-import { applyVariations, hasPaletteColor, deprecatedColorValue } from './utils'
+import {
+  Attention as AttentionIcon,
+  Information as InformationIcon,
+  Success as SuccessIcon,
+  Warning as WarningIcon
+} from 'pcln-icons'
+import {
+  applyVariations,
+  hasPaletteColor,
+  deprecatedColorValue,
+  deprecatedPropType
+} from './utils'
 
 const bannerColors = {
   green: {
     backgroundColor: 'green',
     color: 'white',
```

Comment:
should we remove the old color use?

Reply:
We should but I don't think that requires breaking change, so it need not block this. @achoboter agree?

Reply:
cc @unscsprt